Content: Remnant Chapter 2 Cognizance part II (#6392)
endless-sky/endless-sky: data/map.txt

	government Uninhabited
	security 0

planet Baianus
	attributes uninhabited
	landscape land/canyon12

> 	description `This frozen water world is reminiscent of Titan back in Sol: The surface is little more than a shattered wasteland of ice and the ocassional rock outcropping, but the unbreathably thin atmosphere contains countless tantilizing traces of chemicals that suggest that something more might exist below the surface.`

Reviewer: ```suggestion
	description `This frozen water world is reminiscent of Titan back in Sol: The surface is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
```

		threshold 3000
		fleet "Small Deep Security" 14

planet Fertriery
	attributes "requires: gaslining"
	landscape land/nasa8

> 	description `Fertriery is a fairly typical world with a particularly dense atmosphere with significant levels of chlorine and fluorine. The only unusual factor is the above-average temperatures, likely due to the decreased energy loss into the surrounding space.`

Reviewer 2: ```suggestion
	description `Fertriery is a fairly typical world with a particularly dense atmosphere that includes significant levels of chlorine and fluorine. The only unusual factor is the above-average temperatures, likely due to the decreased energy loss into the surrounding space.`
```
Just trying to avoid the double usage of `with`.

		threshold 3000
		fleet "Small Deep Security" 14

planet Fertriery
	attributes deep station
	landscape land/clouds_00

> 	description `It is unclear exactly what Fertriery is. While it does orbit Postverta, and it has sufficent pressure to threaten most ships. Despite that pressure and the gravity it implies, Fertriery lacks the cohesion or distinct shape normally expected of a planet. Preliminary sensor sweeps have been been stymied by inconsistent readings. Given Fertriery's history, the Remnant are disinclined to rule out even the most far-fetched idea as to what it is or how it exists.`

Reviewer: ```suggestion
	description `It is unclear exactly what Fertriery is. While it does orbit Postverta, and it has sufficient pressure to threaten most ships. Despite that pressure and the gravity it implies, Fertriery lacks the cohesion or distinct shape normally expected of a planet. Preliminary sensor sweeps have been been stymied by inconsistent readings. Given Fertriery's history, the Remnant are disinclined to rule out even the most far-fetched idea as to what it is or how it exists.`
```
